pkgs/by-name/ka/kafka-cmak/package.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  unzip,
+  jdk,
+  gawk,
+  makeWrapper,
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  name = "CMAK";
+  version = "3.0.0.6";
+
+  src = fetchurl {
+    url = "https://github.com/yahoo/CMAK/releases/latest/download/cmak-${finalAttrs.version}.zip";
+    hash = "sha256-PolFcvFagDTR1CUee2gVXdxZzjioobiiHWUUCUbCGe8=";
+  };
+
+  buildInputs = [
+    gawk
+    unzip
+    makeWrapper
+    jdk
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv ./* $out
+    wrapProgram $out/bin/cmak \
+      --set JAVA_HOME ${jdk.home} \
+      --prefix PATH : ${lib.makeBinPath [ gawk ]}
+  '';
+
```

Comment:
```suggestion
passthru.updateScript = ./update.sh;
```

Reply:
Lets make sure the bot notice the update.sh script?

Reply:
I've updated it now, nice catch :+1: